.github/workflows/main.yaml

```diff
@@ -8,7 +8,7 @@ on:
     branches:
       - main
   schedule:
-    - cron: "0 8 * * 4"
+    - cron: "0 6 * * 0"
```

Comment:
Maybe it would make sense to use slightly different schedules for different projects? A few minutes difference would probably help with runner availability.